src/Test.UnitTests.Sarif.Multitool.Library/SuppressCommandTests.cs

```diff
@@ -167,12 +167,12 @@ private static void VerifySuppressCommand(SuppressOptions options)
 
             if (options.Timestamps && suppression.TryGetProperty("timeUtc", out DateTime timeUtc))
             {
-                timeUtc.Should().BeCloseTo(DateTime.UtcNow, DateTimeAssertPrecision);
+                timeUtc.Should().BeCloseTo(DateTime.UtcNow, TimeSpan.FromMilliseconds(DateTimeAssertPrecision));
             }
 
             if (options.ExpiryInDays > 0 && suppression.TryGetProperty("expiryUtc", out DateTime expiryUtc))
             {
-                expiryUtc.Should().BeCloseTo(DateTime.UtcNow.AddDays(options.ExpiryInDays), DateTimeAssertPrecision);
+                expiryUtc.Should().BeCloseTo(DateTime.UtcNow.AddDays(options.ExpiryInDays), TimeSpan.FromMilliseconds(DateTimeAssertPrecision));
```

Comment:
>TimeSpan.FromMilliseconds(DateTimeAssertPrecision) [](http://example.com/codeflow?start=92&length=50)

I must have done the upgrade for other projects before I remember the same change, this is correct.

Reply:
Thanks for the confirmation.